examples/seismic/model.py

```diff
@@ -126,6 +126,34 @@ def demo_model(preset, **kwargs):
                      dtype=dtype, spacing=spacing, nbpml=nbpml, epsilon=epsilon,
                      delta=delta, theta=theta, phi=phi, **kwargs)
 
+    elif preset.lower() in ['layers-tti-noazimuth', 'twolayer-tti-noazimuth',
```

Comment:
what's this new one?

Reply:
no azimuth angle. Only rotation in `xz` plane not in y.

Reply:
remind me (even offline if you want) what's the difference with vti?